oslo/torch/nn/parallel/data_parallel/zero/utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+from oslo.torch.nn.parallel.data_parallel.zero.utils.heterogeneous_hook import (
+    HeterogeneousZeROHook,
+)
+
+__ALL__ = ["HeterogeneousZeROHook"]
```

Comment:
Should we use string here?

Reply:
Anyway, @KKIEEK other code looks great.

Reply:
> Should we use string here?

As far as I know, using a list of strings to assign the `__all__` variable is more common, as mentioned in [this link](https://docs.python.org/3/tutorial/modules.html). However, if you don't prefer it, it can be later modified according to the convention of `oslo` repo.